src/celeritas/optical/GeneratorDistributionData.hh

```diff
@@ -16,11 +16,13 @@
 
 namespace celeritas
 {
+namespace optical
+{
 //---------------------------------------------------------------------------//
 /*!
  * Pre- and post-step data for sampling optical photons.
  */
-struct OpticalStepData
+struct DispatcherStepData
```

Comment:
Do you think we should call this `GeneratorStepData` since it's stored in the `GeneratorDistributionData` and used by the optical generators?